configure.ac

```diff
@@ -88,6 +88,17 @@ AS_IF([test "$YACC" != "bison -y"], [AC_MSG_ERROR([bison not found but required]
 
 PKG_PROG_PKG_CONFIG
 
+# PKG_CHECK_VAR added to pkg-config 0.28
+m4_define_default(
+    [PKG_CHECK_VAR],
+    [AC_ARG_VAR([$1], [value of $3 for $2, overriding pkg-config])
+     AS_IF([test -z "$$1"], [$1=`$PKG_CONFIG --variable="$3" "$2"`])
+     AS_IF([test -n "$$1"], [$4], [$5])])
+
+PKG_CHECK_VAR(bashcompdir, [bash-completion], [completionsdir], ,
+  bashcompdir="${sysconfdir}/bash_completion.d")
+AC_SUBST(bashcompdir)
```

Comment:
Actually, if you do `configure --help` you'll see `bashcompdir` listed as an environment variable you can use to override the `completionsdir` variable from `bash-completion.pc`.
```
$ ./configure --help
...
Some influential environment variables:
...
  bashcompdir  value of completionsdir for bash-completion, overriding pkg-config
...
```
Would that suffice for you?


(I'll capitalize/rename the variable so it's consistent with the other environment variables listed there.)